cmd/soroban-rpc/internal/events/cursor_test.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func TestParseCursor(t *testing.T) {
-	for _, cursor := range []Cursor{
+	for _, cursor := range []*Cursor{
```

Comment:
this doesn't seem necessary?

Reply:
@Shaptic There were some receivers in https://github.com/stellar/soroban-rpc/blob/main/cmd/soroban-rpc/internal/events/cursor.go#L45 that use pointer and some dont. For sake of uniformity, I converted all receivers to pointers. Do not know if that is a bad idea. 

Maybe@tamirms @2opremio have any suggestions?

Reply:
a Cursor is an immutable type so it should not be using pointer receivers except for UnmarshalJSON(). I don't think there should be any changes to cmd/soroban-rpc/internal/events/cursor.go